e2e/login.e2e-spec.ts

```diff
@@ -0,0 +1,133 @@
+import type { INestApplication } from '@nestjs/common';
+import { HttpStatus } from '@nestjs/common';
+import { Test } from '@nestjs/testing';
+import * as pactum from 'pactum';
+
+import { adminStub, loginInputValidPasswordDtoStub } from '../src/admin/stubs';
+import { AppModule } from '../src/app.module';
+import { PrismaService } from '../src/prisma/prisma.service';
+
+function graphqlRequest(query: string) {
```

Comment:
we have this function in two place (login.e2e and smoke.e2e)
it's better to extract it to another file.